app/src/main/java/com/shagalalab/constitution/ui/base/SearchableFragment.kt

```diff
@@ -40,4 +45,17 @@ open class SearchableFragment(resId: Int) : Fragment(resId) {
         )
         super.onCreateOptionsMenu(menu, inflater)
     }
+
+    private fun changeSearchViewTextColor(view: View?) {
+        if (view != null) {
+            if (view is TextView) {
+                view.setTextColor(Color.WHITE)
+                return
+            } else if (view is ViewGroup) {
+                for (i in 0 until view.childCount) {
+                    changeSearchViewTextColor(view.getChildAt(i))
+                }
+            }
+        }
+    }
```

Comment:
I am still wondering why we need this code?

Reply:
Are you asking about "changeSearchViewTextColor" function? I need it to find textView in SeachView and change text color.

Reply:
That's a very inefficient way of doing it (because you are traversing the view hierarchy, and most probably with recursion), you should be able to achieve the same results with themes and styles. Try doing some research there.